apps/src/templates/sectionsRefresh/CurriculumQuickAssign.jsx

```diff
@@ -119,24 +122,13 @@ export default function CurriculumQuickAssign() {
         <QuickAssignTable
           marketingAudience={marketingAudience}
           courseOfferings={courseOfferings}
+          updateCourse={course => updateSection('course', course)}
```

Comment:
I think this assignment won't work until we hook up the version dropdown right? It looks like we'll need a [course_version_id](https://github.com/code-dot-org/code-dot-org/blob/c8f400b88e7b95f7ac180ed11e97aaca06b36295/dashboard/app/controllers/api/v1/sections_controller.rb#L285) for assignment to work

Reply:
Hi! Yes- I am not quite thinking about the save requirements yet. Just making sure I have a clear/operational pipeline to pass through whatever data I eventually discover we will need. Does that make sense?

Reply:
Okay I updated this to be a 'course' object in the section instead, where displayName is only one piece. That way we can easily add whatever else we will need.